.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
   - ruby ./scripts/image_checker.rb
   - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
     "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
+    coala --ci"
```

Comment:
I merged #665 - which made this change -- @yukiisbored -- to resolve this -- just back this change out..when you'll get a conflict -- just take what's currently in `gh-pages` branch in this repo.